wordpress/PeachPied.WordPress.msbuildproj

```diff
@@ -12,7 +12,7 @@
     <Description>WordPress project transformed to managed .NET Standard library.</Description>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**/*.php" Exclude="wp-config-sample.php;wp-content/uploads/**;wp-content/plugins/hello.php;wp-content/plugins/akismet/**" />
+    <Compile Include="**/*.php" Exclude="wp-includes/class-json.php;wp-config-sample.php;wp-content/uploads/**;wp-content/plugins/hello.php;wp-content/plugins/akismet/**" />

```

Comment:
thank for the PR;
@diddledan - is it safe to not compile `class-json.php` at all? Isn't `Services_JSON` used somewhere?

Reply:
Sorry, now I see it is not. It can safely excluded.